pwiz_tools/Skyline/SkylineFiles.cs

```diff
@@ -204,7 +204,68 @@ private void openMenuItem_Click(object sender, EventArgs e)
             }
         }
 
-        public bool OpenSharedFile(string zipPath, FormEx parentWindow = null)
+        /// <summary>

```

Comment:
Support functions go below the main function not above. Above is old-school for C compilers.

Reply:
On consideration, I realize that they're actually in the wrong file altogether - they're not specific to the UI and I've made them static methods of SrmDocument. This will be handy if we want to do something similar for the commandline (though I think scripters can probably be relied on to pass actual .sky files)